src/gui/torrentcontentmodelitem.h

```diff
@@ -35,47 +35,51 @@
 #include <QVariant>
 
 namespace prio {
```

Comment:
Fixing style in cases like this is easier than typing "This code is not mine." :)